src/nvim/options.lua

```diff
@@ -2766,5 +2766,14 @@ return {
       varname='p_wd',
       defaults={if_true={vi=0}}
     },
+    {
+      full_name='livesub', abbreviation='lsu',
+      type='bool', scope={'global'},
+      vi_def=true,
+      redraw={'everything'},
+      varname='p_sub',
+      defaults={if_true={vi=false, vim=false}}
+    },
```

Comment:
I believe the problem is you added this in the wrong spot. Try putting it in alphanumeric order: after the `listchars` entry.